lib/order.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/menu'
+
+class Order
+
+  attr_reader :basket, :menu, :total
+
+  def initialize(menu_klass= Menu)
+    @menu_klass = menu_klass
+    @menu = @menu_klass.new
+    @basket = Hash.new(0)
+    @total = 0
+  end
+
+  def add_order(name, quantity)
+    @basket[name] += quantity
+    basket.each {|dish, qty| "you have added #{dish} x #{qty} in your basket"}
+    @basket
+  end
+
+  def basket_sum
+    raise 'Basket is empty' if basket.empty?
+    message = ""
+    @basket.each {|item, qty| message = message + "#{item} x #{qty} = £#{(menu.dish[item]*qty)}"}
```

Comment:
Metrics/LineLength: Line is too long. [97/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SelfAssignment: Use self-assignment shorthand `+=`. (https://github.com/bbatsov/ruby-style-guide#self-assignment)